ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigBuilder.java

```diff
@@ -701,6 +701,9 @@ public SpecConfigBuilder mergeBuilder(final Consumer<MergeBuilder> consumer) {
     private Bytes32 terminalBlockHash;
     private UInt64 terminalBlockHashActivationEpoch;
 
+    // Optimistic Sync
+    private int safeSlotsToImportOptimistically = 128;
```

Comment:
is this the right place to handle this constant?

Reply:
I think so. It's a slightly odd case because we don't normally have defaults in `SpecConfig` - they all have to be in the config file.  This one is essentially a constant from the spec so won't be in the config file but must be overridable on the CLI (which I'm yet to add). We don't want the default to be in the CLI Options class as then it wouldn't be set when creating Teku through the programmatic API Anton added so I think it winds up here.

We might wind up with a constant for the default value when the CLI option comes in. So I'm going to leave it here for now and see how things pan out when adding the CLI option.